resources/build/networking/wired.py

```diff
@@ -63,6 +63,12 @@ def _on_model(self) -> None:
                 support.BuildSupport(self.model, self.constants, self.config).enable_kext("nForceEthernet.kext", self.constants.nforce_version, self.constants.nforce_path)
             elif isinstance(controller, device_probe.Marvell) or isinstance(controller, device_probe.SysKonnect):
                 support.BuildSupport(self.model, self.constants, self.config).enable_kext("MarvelYukonEthernet.kext", self.constants.marvel_version, self.constants.marvel_path)
+            
+            # Pre-Ivy Bridge Aquantia Ethernet Patch
+            if isinstance(controller, device_probe.Aquantia) and controller.chipset == device_probe.Aquantia.Chipsets.AppleEthernetAquantiaAqtion:
+                if smbios_data.smbios_dictionary[self.model]["CPU Generation"] < cpu_data.CPUGen.ivy_bridge.value:
```

Comment:
Edge case scenario but would recommend adding a model check first to ensure there's not a key error (ex. some unknown model runs like an AppleInternal unit, better to not crash even if we don't support it)

```py
if self.model in smbios_data.smbios_dictionary:
```